From 16c1833ade4c036b30b8761d2c4a5bd85cc65c44 Mon Sep 17 00:00:00 2001
From: Robert Yang <liezhi.yang@windriver.com>
Date: Tue, 8 Jan 2019 06:27:06 +0000
Subject: [PATCH] Disable the check for different filesystems

The logrotate supports rotate log across different filesystems now, so
disable the check for different filesystems.

Upstream-Status: Pending

Signed-off-by: Robert Yang <liezhi.yang@windriver.com>

---
 config.c | 9 ---------
 1 file changed, 9 deletions(-)

diff --git a/config.c b/config.c
index d2488f1..1de3745 100644
--- a/config.c
+++ b/config.c
@@ -1902,15 +1902,6 @@ duperror:
                             }
 
                             free(ld);
-
-                            if (sb.st_dev != sb2.st_dev
-                                    && !(newlog->flags & (LOG_FLAG_COPYTRUNCATE | LOG_FLAG_COPY | LOG_FLAG_TMPFILENAME))) {
-                                message(MESS_ERROR,
-                                        "%s:%d olddir %s and log file %s "
-                                        "are on different devices\n", configFile,
-                                        lineNum, newlog->oldDir, newlog->files[j]);
-                                goto error;
-                            }
                         }
                     }
 
